seshat-node/native/src/tasks.rs

```diff
@@ -539,3 +539,32 @@ impl Task for SetUserVersionTask {
         }
     }
 }
+
+pub(crate) struct ShutDownReindexTask(pub(crate) Mutex<Option<RecoveryDatabase>>);
+
+impl Task for ShutDownReindexTask {
```

Comment:
Hmm, does it actually cause reindexing...? It seems to only shutdown...? Perhaps better to rename.

Reply:
Right, should have named it `ShutDownRecoveryTask` since it's for the revovery db.